<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDashboard</class>
 <widget class="QWidget" name="ClientDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>734</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ClientDashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QTableWidget" name="tableFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>801</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>File Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Upload Date</string>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File Size</string>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editDownPath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); 
border-radius: 20px;
padding-left: 20px;
padding-right: 20px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDownBrowse">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonDownBrowse{ 
border-radius: 20%;
background-color: white;
}
#buttonDownBrowse:hover{
border-radius: 20%;
background-color: rgb(177,177,177);
}
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDownload">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonDownload{ 
border-radius: 20%;
background-color: white;
}
#buttonDownload:hover{
border-radius: 20%;
background-color: rgb(177,177,177);
}
</string>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUpFilename">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>560</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); 
border-radius: 20px;
padding-left: 20px;
padding-right: 20px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpBrowse">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonUpBrowse{ 
border-radius: 20%;
background-color: white;
}
#buttonUpBrowse:hover{
border-radius: 20%;
background-color: rgb(177,177,177);
}
</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUpPath">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); 
border-radius: 20px;
padding-left: 20px;
padding-right: 20px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpload">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>620</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonUpload{
border-radius: 20%;
background-color: white;
}
#buttonUpload:hover{
border-radius: 20%;
background-color: rgb(177,177,177);
}
</string>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>570</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSearch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255); 
border-radius: 20px;
padding-left: 20px;
padding-right: 20px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSearch">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonSearch{ 
border-radius: 20%;
background-color: white;
}
#buttonSearch:hover{
border-radius: 20%;
background-color: rgb(177,177,177);
}
</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRefresh">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buttonRefresh{ 
border-radius: 20%;
background-color: white;
}
#buttonRefresh:hover{
border-radius: 20%;
background-color: rgb(177,177,177);
}
</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/filelist);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>15</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>User Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>151</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>151</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/control);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>570</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/file);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/download);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>485</y>
     <width>168</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/upload);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>485</y>
     <width>151</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>831</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Status...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
